internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -156,6 +206,16 @@ func shouldIRQLoadBalancingBeDisabled(annotations fields.Set) bool {
 		annotations[crioannotations.IRQLoadBalancingAnnotation] == annotationDisable
 }
 
+func shouldCStatesBeConfigured(annotations fields.Set) (present bool, value string) {
+	value, present = annotations[crioannotations.CPUCStatesAnnotation]
```

Comment:
I don't really like how the two values are swapped; perhaps it's better to return it in the same order, so the user would do

```go
if value, configure := shouldCStatesBeConfigured(s.Annotations()); configure {
   ....
```

With that change implemented, maybe we don't need this function at all, and just have
```go
if value, ok := s.Annotations()[crioannotations.CPUCStatesAnnotation]; ok {
   ...
```

but if you think that having this as a separate function improves readability or enables better code reuse, please feel free to leave it.

Reply:
I did this on purpose to be consistent with the other functions that were already here (e.g. shouldCPUQuotaBeDisabled). The existing functions are returning a similar bool as their first (and only parameter) so I felt returning the bool as the first parameter was more consistent. That makes sense (at least in my mind) so I'd prefer to leave these as they are.